### ISSUE TITLE
[Feature]: Please add data export for free space to bazooka or similar addons.

### ISSUE THREAD

**Issue by fsdraconis** [Cidan/BetterBags#675]
### Describe the feature

I use bazooka as a data display bar at the top of my screen. Previously I was using adibags and it had the feature to show how much free space I had available this way. 

### Describe the implementation

To have an option to enable data export to bazooka. Bazooka is a data broker addon. I would like to enable to see the used and free bag space for both normal and resource bag.

**Comment by Cidan** [comment]
AdiBags never integrated with bazooka. It's likely bazooka dug into AdiBags. Please reach out to the Bazooka devs for this.

**Comment by fsdraconis** [comment]
Bazooka is just a data broker display app. It can not function with out the other side of things pushing data as a plug in. Similar to Titan and other data broker display tools. I can't say for certain but it may have been a plugin for adibags that someone made. Bazooka does not have the means to pull data on its own. It acts like a tool box waiting for tools to be put into it. It has zero features on its own. But that is fine if you are not interested in adding this feature. It was just handy to not have to open my bags just to see how much room I had left. Thanks for your time.

**Comment by fsdraconis** [comment]
When I went looking for additional means to solve this after the last post I sent. I ran into Broker Everything also. It actually solved a couple things for me besides the bag interest. So that worked out well. Thank you for looking into it. I'm playing on a 5120x1440 monitor. So I have three different groups for different broker data. Most of it is just there to glance at. Some of it like the guild/friends is mouse over. A few other addons with support for it. I ended up disabling a few things from broker everything that just seemed to be more than I really needed. Thank you again for looking into this further.
